saleor/graphql/context.py

```diff
@@ -58,6 +59,12 @@ def set_app_on_context(request):
 def get_user(request: RequestWithUser) -> Optional[UserType]:
     if not hasattr(request, "_cached_user"):
         request._cached_user = cast(UserType, authenticate(request=request))
+        try:
+            user_id = request._cached_user.id
+        except AttributeError:
+            pass
```

Comment:
In which cases would this exception happen?

Reply:
Can't you just do `getattr`? But it does look like there `id` property available for use: https://github.com/django/django/blob/0dd29209091280ccf34e07c9468746c396b7778e/django/contrib/auth/models.py#L417-L418

Reply:
strange, I will double check